### PR TITLE
add the media-kit page.

### LINKED ISSUE #78: Create the "Media Kit" page
Previous website source: https://github.com/gitpod-io/website/blob/master/src/pages/media-kit.tsx
Previous website link: https://www.gitpod.io/media-kit/

Note: There is no Figma design. Please follow the previous website guideline with the new branding applied.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #78 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/447"></a>

### CODE REVIEW

File: src/components/media-kit/logo-section.svelte [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  ];
+</script>
+
+<section class="text-center">
+  <div class="max-w-3xl m-auto mb-small">
+    <h2>About</h2>
+    <p>
+      Please don’t modify the logos or use them in any altered way.
+      <br />
+      For more details <a href="/assets/style-guide.pdf">see style guide</a>.
```
Comment (nisarhassan12) [inline src/components/media-kit/logo-section.svelte:33]: @mikenikles This link is behaving quite oddly when I click through it gives a 404 but when I visit the url http://localhost:3000/assets/style-guide.pdf directly it gives me the desried result.  

https://user-images.githubusercontent.com/46004116/118410012-c15f3280-b6a6-11eb-9f5b-4bf6fefea6d8.mp4

**Comment by ChristinFrohne** [comment]
Great thanks! Some comments

- [ ] please add a link to the footer under "company"
- [ ] we need a consistent naming. Currently, the link says "media kit" but on the page, it says "press kit". Let's go with "media-kit" I'd say that sounds more generic
- [ ] having two headlines directly after each other looks a bit weird. I added an image to the mockup. If you agree, please add it to the page
- [ ] the style guide should open in a new tab
- [ ] update color for "sand light" (# F5F4F4 -see mockup)
- [ ] please contact Johannes to get a better profile image with a higher resolution 
- [ ] on mobile the buttons are misaligned 
Bildschirmfoto 2021-05-17 um 11 38 43

File: src/assets/scss/_buttons.scss [hunk 2]
```diff
@@ -96,7 +96,8 @@ a {
 }
 
 .btn-otherbrand {
-  display: flex;
+  display: inline flex;
+  min-width: 5rem;
```
Comment (mikenikles) [inline src/assets/scss/_buttons.scss:100]: **question**: This impacts all buttons with that class name. Is it a safe change that doesn't impact the look & feel of these buttons?

File: src/components/section.svelte [hunk 3]
```diff
@@ -1,5 +1,6 @@
-<script>
-  export let id;
+<script lang="ts">
+  export let id: string;
+  export let className: string;
```
Comment (mikenikles) [inline src/components/section.svelte:3]: We have to be careful when we add mandatory props like this. For all existing `<Section />` components, we now have an empty space in the `class` attribute and get the following warning in the browser console:

```
<Section> was created without expected prop 'className'
```

If `className` is optional, we can mark it as such by giving it a default value.

Remember, it is important to review and address warnings displayed in the terminal and the browser console.

File: src/components/media-kit/logo-section.svelte (hunk 1 above)
Comment (mikenikles) [reply src/components/media-kit/logo-section.svelte:33]: I can't reproduce that on the [PR preview URL](https://deploy-preview-447--gitpod-kumquat.netlify.app/media-kit), can you? Tested in Safari, Chrome, Brave and Firefox.

In any case, please have a look at the SvelteKit documentation with an explanation of why you likely see this and how to fix it: https://kit.svelte.dev/docs#anchor-options-rel-external

**Review by mikenikles — CHANGES_REQUESTED** [review]

File: src/components/media-kit/logo-section.svelte (hunk 1 above)
Comment (nisarhassan12) [reply src/components/media-kit/logo-section.svelte:33]: Thanks. I can't reproduce this as well on the deploy preview as well. It only happend on `localhost`.

File: src/assets/scss/_buttons.scss (hunk 2 above)
Comment (nisarhassan12) [reply src/assets/scss/_buttons.scss:100]: Thanks. @mikenikles I tested all the instances and it doesn't impact the look and feel of them. Now that I have made the buttons intact with the mockup it is no longer required.

**Review by mikenikles — APPROVED** [review]

**Pull request merged**